client-vite/README.md

```diff
@@ -0,0 +1,35 @@
+# Arbeidsflate-frontend
```

Comment:
I suggest calling the file `Readme.md`.

Maybe remove stuff (like eslint section) that we may not need.